### DISCUSSION TITLE
Modelling two sets of multidimensional data

### DISCUSSION THREAD

**Discussion by lovecodng** [meta-pytorch/botorch#813] [Q&A] [answered]
Is it possible to use two set of multidimensional data instead of one I mean one set for each objective function, For instance, if the first objective function is fitted using data x with dimension (n*d), then the second objective function is fitted using data x multiplied by constant array (c) with dimension (d), such as x*c.

**Comment by saitcakmak** [comment, marked as answer]
Answer from the other thread: https://github.com/pytorch/botorch/discussions/809#discussioncomment-818237
